doc/release_notes/v2.1.0.rst

```diff
@@ -39,13 +39,39 @@ Enhancements
     *Waveform Sequence* element.
 * When JPEG 2000 image data is unsigned and the *Pixel Representation* is 1
   the image data is converted to signed (:issue:`1149`)
+* Added :attr:`~pydicom.uid.AllTransferSyntaxes`
 
 Changes
 .......
 * Reading and adding unknown non-private tags now does not raise an exception
   per default, only when :attr:`~pydicom.config.enforce_valid_values` is set
   (:issue:`1161`)
 * Data dictionaries updated to version 2020c of the DICOM Standard
+* The following UID constants are deprecated and will be removed in v2.2:
+
+  * ``JPEGBaseline`` (use :attr:`~pydicom.uid.JPEGBaseline8Bit` instead)
+  * ``JPEGExtended`` (use :attr:`~pydicom.uid.JPEGExtended12Bit` instead)
+  * ``JPEGLossless`` (use :attr:`~pydicom.uid.JPEGLosslessSV1` instead)
+  * ``JPEGLSLossy`` (use :attr:`~pydicom.uid.JPEGLSNearLossless` instead)
+  * ``JPEG2000MultiComponentLossless`` (use
+    :attr:`~pydicom.uid.JPEG2000MCLossless` instead)
+  * ``JPEG2000MultiComponent`` (use :attr:`~pydicom.uid.JPEG2000MC` instead)
+
+* Starting in v2.2 the value for ``JPEGLossless`` will change from
```

Comment:
I don't like this much, as it may introduce bugs in user code (not everyone checks the release notes), but I don't have a better idea... At least the problem will note happen if one updates to 2.1 before the update to 2.2, so I guess it is ok.

Reply:
Me either, but I think however painful it is, it'll have to happen at some point.

Reply:
Can we alias with deprecation? Still not easy to do, perhaps.

Reply:
Wait, never-mind, actually a UID change.

Reply:
Can do it in Python 3.7+, will update shortly

Reply:
[PEP562](https://www.python.org/dev/peps/pep-0562/): Module `__getattr__`